sf-fx-runtime-java-logger/src/test/java/com/salesforce/functions/jvm/runtime/logger/DefaultLoggingFormatterTest.java

```diff
@@ -30,16 +31,21 @@ public void clearAndSetMDC() {
 
   @Test
   public void test_1() {
+    String localDateTimeString = LocalDateTime.now(ZoneOffset.UTC).format(DATE_TIME_FORMATTER);
     String result = formatter.format("foo.bar.baz", Level.DEBUG, "This is a message!");
+
     assertThat(
         result,
         is(
             equalTo(
-                "00:00:00.000 DEBUG [INVOCATION e3a4ae2b-fefb-4277-89d0-7068e7e39b99] foo.bar.baz - This is a message!\n")));
+                String.format(
+                    "UTC=%s level=DEBUG loggerName=foo.bar.baz message=This is a message! invocationId=e3a4ae2b-fefb-4277-89d0-7068e7e39b99 \n",
```

Comment:
The `message` field is still not quoted and the appropriate test for required tests inside the quotes is missing. This applies to all other tests as well.